public/locations-list-map.js

```diff
@@ -1112,16 +1112,21 @@ function number_format(number, decimal_places, dec_seperator, thou_seperator) {
     }
     f[1] = g;
   }
-  if (thou_seperator != '' && f[0].length > 3) {
+  if (thou_separator != '' && f[0].length > 3) {
     let h = f[0];
     f[0] = '';
     for (let j = 3; j < h.length; j += 3) {
       let i = h.slice(h.length - j, h.length - j + 3);
-      f[0] = thou_seperator + i + f[0] + '';
+      f[0] = thou_separator + i + f[0] + '';
     }
     let j = h.substr(0, (h.length % 3 == 0) ? 3 : (h.length % 3));
     f[0] = j + f[0];
   }
-  dec_seperator = (decimal_places <= 0) ? '' : dec_seperator;
-  return f[0] + dec_seperator + f[1];
+  dec_separator = (decimal_places <= 0) ? '' : dec_separator;
+  return f[0] + dec_separator + f[1];
+}
+
+window.onload = function(){
```

Comment:
Apologies if this was talked about in your original PR but -- I think this code can go here instead:

https://github.com/findthemasks/findthemasks/blob/master/public/locations-list-map.js#L374-L389

that block of code is already within a `$(document).ready` call and, within that, inside an `i18n:ready` event, so it would ensure that `i18n` is fully loaded by the time it tries to execute. It's possible your code *could* fail (though obviously it's working now when I'm testing it).

Reply:
I didn't notice that -- I'll change it.